pkg/installation/installation.go

```diff
@@ -368,13 +368,13 @@ func (i *Installation) applyOverrideFiles() error {
 		if err != nil {
 			fmt.Printf("unable to open file: %s. error: %s\n",
```

Comment:
If returning the error, no need to print it here.

Just wrap it and return so that the installation step can print it and fail properly.